test/smoke/src/main.ts

```diff
@@ -279,7 +279,10 @@ after(async function () {
 	if (opts.log) {
 		const logsDir = path.join(userDataDir, 'logs');
 		const destLogsDir = path.join(path.dirname(opts.log), 'logs');
-		await new Promise((c, e) => ncp(logsDir, destLogsDir, err => err ? e(err) : c(undefined)));
+		// {{ SQL CARBON EDIT }} Checks for the existence of the logs directory before accessing.
```

Comment:
It would be more helpful to indicate why we need to do this check (the code is clear enough that it's doing this check).

 Did you ever figure that out?

Reply:
From what I've seen, the error is occurring because we don't ever create the "/tmp/t.../d/logs" directory. I followed the usages of tmpDir, and we explicitly create things like the "extensions-dir" under "/tmp/t../", but we don't do the same for "/d/logs", I'm thinking that it's safe to remove that call. I will remove and kick off a new build to see what we get.